### PR TITLE
Bug 1325342: Updated Disabling Swap Memory section

### PR DESCRIPTION
**Pull request by ahardin-rh**
https://bugzilla.redhat.com/show_bug.cgi?id=1325342

### CODE REVIEW

**Comment by ahardin-rh** [comment]
@adellape @tnguyen-rh @sferich888 PTAL

File: install_config/install/prerequisites.adoc [hunk 1]
```diff
@@ -121,6 +121,14 @@ Therefore, the recommended size of a cluster of 2000 pods would be 20 CPU cores
 and 50 GB of RAM, as well as the minimum requirements for a master host of 2 CPU
 cores and 8 GB of RAM.
 
+[NOTE]
+====
+It is important to understand that oversubscribing the physical resources on a
```
Comment (tnguyen-rh) [inline install_config/install/prerequisites.adoc]: The wording is awkward to my ear (although i can see that the text is similar to that above, which does not sound awkward, strangely enough).  Suggest changing the admonition to `IMPORTANT`, and chopping the blurb head to start w/ "Oversubscribing the physical resources...".

**Comment by tnguyen-rh** [comment]
@ahardin-rh One quibble, otherwise LGTM.

**Comment by ahardin-rh** [comment]
[rev_history]
|link:../admin_guide/overcommit.html[Overcommitting]
|Updated the link:../admin_guide/overcommit.html#disabling-swap-memory[Disabling Swap Memory] section with options that can help users avoid having to swap and added a Warning box stating that disabling swap memory is not recommended.
%
|link:../install_config/install/prerequisites.html[Installing -> Prerequisites]
|Added an Important box to the link:../install_config/install/prerequisites.html#sizing-recommendations[Sizing Recommendations] section advising that oversubscribing the physical resources on a node affects resource guarantees the Kubernetes scheduler makes during pod placement.
%

**Pull request merged**